scss/inc/_forms.scss

```diff
@@ -410,11 +410,14 @@ label {
             left: -1em;
         }
     }
+    input[type="radio"]:disabled,
+    input[type="checkbox"]:disabled {
+        opacity: 0;
+    }
     input[type="radio"],
     input[type="checkbox"] {
-        visibility: hidden;
-        pointer-events: none;
         position: absolute;
+        opacity: 0;
```

Comment:
I'm still not 100% happy with this since it still takes space and is clickable (yes, an edge case)
Maybe see `visually-hidden` from LDS https://github.com/oat-sa/live-design-system/blob/master/packages/identity/css/mixins/_statuses.css#L43 ?
What do you think?

https://user-images.githubusercontent.com/38751932/147256403-2b4f1884-1598-408f-a118-a8036f0f5da1.mp4

Reply:
Yeah. Looks weird. I'll change this.

Reply:
Convert approach to LDS-similar, as you suggested.